src/cdk/utils/create-unauthorized-gateway-response.ts

```diff
@@ -11,13 +11,27 @@ export function createUnauthorizedGatewayResponse(
     return;
   }
 
+  const responseParameters: Record<string, string> = {
+    'gatewayresponse.header.WWW-Authenticate': "'Basic'",
+  };
+
+  const {unauthorizedResponseHeaders} = stackConfig.basicAuthenticationConfig;
+
+  if (unauthorizedResponseHeaders) {
+    const {accessControlAllowOrigin} = unauthorizedResponseHeaders;
+
+    responseParameters[
+      'gatewayresponse.header.Access-Control-Allow-Origin'
+    ] = `'${accessControlAllowOrigin}'`;
```

Comment:
maybe check if accessControlAllowOrigin is undefined

Reply:
yes, that was a brainfart